### PR TITLE
Backport [9.4.X] : add fastsim 2017 PU and pmx wfs

### PR DESCRIPTION
**Pull request by zhenhu**
backport of #24829 

Note these newly added PU and pmx workflows can not be tested currently. 
The backport of fastSim2017 code (#24624) had not been merged when 9_4_10 was built, so the "RelValMinBiasFS_13_UP17_ForMixing" dataset can not be produced in 9_4_X yet.  

Two options now: 
1. We can merge this PR now (without testing these new wfs, which might be dangerous). Once 9_4_11_cand1 is built, we will produce the missing MinBias dataset and test the wfs. 
2. We build 9_4_11_cand1 without this PR, and produce the MinBias dataset with 9_4_11_cand1. At that time, this PR can be tested and it will merged for the next release.

### CODE REVIEW

File: Configuration/PyReleaseValidation/python/relval_steps.py [hunk 1]
```diff
@@ -446,6 +446,8 @@ def identitySim(wf):
     'CMSSW_8_1_0_pre9_Geant4102-81X_mcRun2cosmics_startup_peak_v2-v1', # 12 - GEN-SIM input for 1307 cosmics wf from 810_p2
     'CMSSW_9_4_0_pre3-94X_mc2017_realistic_v4-v1',     # 13 - 13 TeV samples with GEN-SIM from PhaseI upgrade
     'CMSSW_9_4_0_pre3-PU25ns_94X_mc2017_realistic_v4-v1',     # 14 - fullSim PU 25ns UP17 premix
+    'CMSSW_9_4_10-94X_mc2017_realistic_v15_FastSim-v1',    # 15 - fastSim MinBias for mixing UP17
+    'CMSSW_9_4_10-PU25ns_94X_mc2017_realistic_v15_FastSim-v1',# 16 - fastSim premixed MinBias UP17
```
Comment (zhenhu) [inline Configuration/PyReleaseValidation/python/relval_steps.py]: this two lines need to be updated to CMSSW_9_4_11_cand1-

**Comment by zhenhu** [comment]
please test

**Comment by kpedro88** [comment]
This backport should be updated with the additional premixing workflow 250402.171 and the necessary components

**Comment by zhenhu** [comment]
> This backport should be updated with the additional premixing workflow 250402.171 and the necessary components

Cand2 is about to built. So maybe it's good to merge the current version to cand2 first, and then we will add the additional workflow 250402.171.

**Comment by zhenhu** [comment]
+1

**Comment by fabiocos** [comment]
@zhenhu @kpedro88 @smuzaffar merging this PR to build 9_4_11_cand2 will introduce a number of artificial failures in the 9_4_X IB until the new samples are produced and made available. Anyway I think that might be acceptable, as it is an understood issue, provided it does not last for too long

**Comment by fabiocos** [comment]
@zhenhu as @kpedro88 makes me notice, this PR is not the exact backport of #24829: workflow 250402.171 seems missing, and you use PUFS50 instead of PUFSAVE50. Why are there differences, and we cannot just have a clean backport?

**Comment by zhenhu** [comment]
Hi @fabiocos and @kpedro88 , the change from PUFSAVE50 to PUFS50 came from a commit after my backport. 
New workflow 250402.171 was added in the original PR is because a similar old workflow 250402.1 in 10_3_X. But in the 9_4_X code, 250402.1 is not even there.

**Comment by zhenhu** [comment]
OK, I can close this PR and remake a clean backport with all features similar as the current version of #24829 .

**Comment by zhenhu** [comment]
This PR has been replaced with a better version: PR #24879

**Pull request closed without merging**